packages/@ionic/cli-utils/src/lib/project/angular/index.ts

```diff
@@ -89,75 +89,26 @@ export class Project extends BaseProject {
     return registry;
   }
 
-  async getFrameworkVersion() {
-    const pkgName = '@ionic/angular';
+  async getPackageVersion(pkgName: 'ionicAngular' | 'ionicCore' | 'ionicSchematicsAngular' |
```

Comment:
This class is one of three project classes:
- `src/lib/project/angular/index.ts`
- `src/lib/project/ionic-angular/index.ts`
- `src/lib/project/ionic1/index.ts`

They all extend `BaseProject`, which is in `src/lib/project/index.ts`, and they all have methods like this. Maybe now is a good time to refactor them to use a single utility method?

Also you don't need to use `'ionicAngular' | 'ionicCore' ...` as the string literal, you can just use the package name itself: `'@ionic/angular' | '@ionic/core' ...` 😄